### ISSUE TITLE
[Feature Request] Product attribute

### ISSUE THREAD

**Issue by webchun** [themexpert/digicom#361]
I'd like to request a product attribute function, so we can add different price for different attribute, for example product can have a Regular license and Extended License

**Comment by shamsbd71** [comment]
We have the plane, but not getting enough time. As this is a free and Open source, we have developed the basic software and continuing the development.

some features require more dedicated time and effort. not sure when you can have them, but gradually we will improve them and will add this essential feature.

thanks

**Comment by webchun** [comment]
Hi @shamsbd71 no problem at all. I'm fully understand with this condition. Just a suggestion, maybe you can develop it as a paid plugin, along with the multivendor feature.

**Comment by ledlightcircus** [comment]
Yes totally agree, this would be an awesome addition, and paid plugin is a good idea !
